tests/integration/backends/test_tpp.py

```diff
@@ -943,6 +944,48 @@ def test_isaric_raw_clinical_variables(select_all):
     ]
 
 
+@register_test_for(tpp.open_prompt)
+def test_open_prompt(select_all):
+    results = select_all(
+        OpenPROMPT(
+            Patient_ID=1,
+            CodedEvent_ID="00000",
+            CodeSystemId=0,  # SNOMED CT
+            ConceptId="100000",
+            ConsultationDate="2023-01-01",
+            Consultation_ID=1,
+            NumericValue=1.0,
+        ),
+        OpenPROMPT(
+            Patient_ID=2,
+            CodedEvent_ID="00000",
+            CodeSystemId=2,  # CTV3 "Y"
+            ConceptId="Y0000",
```

Comment:
According to my reading of the PR description, `ConceptId` should match `CodedEvent_Id` here.

Reply:
It should, thank you for spotting this. Because we're testing that there isn't a `snomedct_code` when `ctv3_code` is a "Y" code, `CodedEvent_ID` matches `ConceptId`. See c772727.